examples/named-views/pages/index.vue

```diff
@@ -0,0 +1,68 @@
+<template lang="html">
+  <div class="main">
+    <h1>Hello there</h1>
+    <p>This is an example of a named views</p>
+    <ul>
+      <li>
+        <nuxt-link to="/">
+          Main
+        </nuxt-link>
+      </li>
+      <li>
+        <nuxt-link to="/section">
+          Section
+        </nuxt-link>
+      </li>
+      <li>
+        <nuxt-link to="/child/123">
+          Child 123
+        </nuxt-link>
+      </li>
+      <li>
+        <nuxt-link to="/child/234">
+          Child 234
+        </nuxt-link>
+      </li>
+    </ul>
+
+    <hr>
+    <div>
+      <div class="left">
+        <router-view name="left" />
```

Comment:
Why not also supporting `<nuxt-child name="left"/>` here?

Reply:
You are right!